### ISSUE TITLE
Error from fresh install

### ISSUE THREAD

**Issue by home-directory** [mbolli/nfsen-ng#13]
![capture](https://user-images.githubusercontent.com/6114636/47444875-ccd07280-d7b7-11e8-824f-1af891d990a2.PNG)

**Comment by mbolli** [comment]
- can apache read the .htaccess file? (file permissions)
- do you have `AllowOverride None` inside your virtualhost configuration? try to change it to `AllowOverride All`.

**Comment by home-directory** [comment]
The thing is if you download the zip source there's no folder config for api.
The folder api is located in nfsen-ng-master\backend\api

**Comment by mbolli** [comment]
Yeah but there’s an Apache `RewriteRule` in the .htaccess file for that.

**Comment by home-directory** [comment]
Ok, this is my conf file as you can see i've AllOverride All, but i've still the same problem
<VirtualHost *:80>
        DocumentRoot /var/www/html/nfsen-ng/frontend/
        #ServerName

        ErrorLog ${APACHE_LOG_DIR}/error.log
        CustomLog ${APACHE_LOG_DIR}/access.log combined

         <Directory /var/www/html/nfsen-ng/frontend>
                Options Indexes Includes FollowSymLinks
                AllowOverride All
                Require all granted
        </Directory>

</VirtualHost>

**Comment by mbolli** [comment]
The .htaccess is in the nfsen-ng directory itself, you probably have to specify `/var/www/html/nfsen-ng/` as `DocumentRoot` and `<Directory>`. Does that work?
